libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -545,6 +545,36 @@ bool AP_AHRS_NavEKF::use_compass(void)
     return AP_AHRS_DCM::use_compass();
 }
 
+// return the quaternion defining the rotation from NED to XYZ (body) axes
+bool AP_AHRS_NavEKF::get_quaternion(Quaternion &quat) const
+{
+    switch (active_EKF_type()) {
+    case EKFType::NONE:
+        return false;
```

Comment:
Should this be giving back the DCM result?


check `get_gyro` for precedent.

Reply:
yes, thanks.  Added, "return AP_AHRS_DCM::get_quaternion(quat);" in a follow up commit.